projects/components/src/list-view/list-view.component.scss

```diff
@@ -7,6 +7,33 @@
   display: flex;
   flex-direction: column;
 
+  .header {
+    flex: 0 0 auto;
```

Comment:
`flex: 0 0 auto;` you probably don't need this if you are setting a fixed width/height

Reply:
Okay, I will check for it.